### ISSUE TITLE
 The POM for com.microsoft.azure:azure-keyvault:jar:1.1 is invalid

### ISSUE THREAD

**Issue by nicenemo** [Azure/azure-keyvault-java#50]
```
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.microsoft.azure:azure-keyvault:jar:1.1 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
```

Tried clearing my maven cache, did not help.

Running with `mvn clean compile -X` gave:

```
[DEBUG] =======================================================================
[WARNING] The POM for com.microsoft.azure:azure-keyvault:jar:1.1 is invalid, transitive dependencies (if any) will not be available: 2 problems were encountered while building the effective model for com.microsoft.azure:azure-keyvault:1.1
[ERROR] 'dependencies.dependency.systemPath' for com.microsoft.azure:azure-keyvault-cryptography:jar must specify an absolute path but is ${basedir}/../azure-keyvault-cryptography/target/azure-keyvault-cryptography-1.1.jar @
[ERROR] 'dependencies.dependency.systemPath' for com.microsoft.azure:azure-keyvault-core:jar must specify an absolute path but is ${basedir}/../azure-keyvault-core/target/azure-keyvault-core-1.1.jar @
```
Maven & Java version:

```
mvn -version

Apache Maven 3.5.3 (3383c37e1f9e9b3bc3df5050c29c8aff9f295297; 2018-02-24T20:49:05+01:00)
Maven home: C:\tools\apache-maven-3.5.3\bin\..
Java version: 1.8.0_181, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.8.0_181\jre
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

Code using this version compiles and works, even after clearing the cache.

**Comment by cheenamalhotra** [comment]
Same Error received with JDK 10.0.2

Java version: 10.0.2, vendor: Oracle Corporation
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

**Comment by schaabs** [comment]
Should be fixed in version 1.1.1
